pages/super-admin/deleted-products/index.tsx

```diff
@@ -0,0 +1,114 @@
+          <div>
+            <h2 className="text-lg font-medium text-custom-color10">Deleted Products</h2>
+            <p className="text-custom-color2 text-sm">List of all deleted products and their details</p>
+          </div>
+          <div>
+            {/* input for search */}
+            <SearchProduct handleSearchChange={handleSubmit} />
+          </div>
+        </div>
+        {/* Deleted products list */}
+        <table className="border-t border-custom-color1 w-full">
+          <thead>
+            {/* Table Headers */}
+            <tr>
+              <th className="text-custom-color2 text-sm font-normal leading-5 px-6 py-6 gap-3 text-left flex  items-center">
+                <input type="checkbox" />
+                <span>Product Name </span>
+                <ArrowDown size="16" className="" />
+              </th>
+              <th className="text-custom-color2 text-sm font-normal leading-5 px-3 py-6 gap-3">Vendor</th>
+              <th className="hidden md:table-cell text-custom-color2 text-sm font-normal leading-5 px-3 py-6 gap-3 ">
+                ID
+              </th>
+              <th className="hidden md:table-cell text-custom-color2 text-sm font-normal leading-5 px-3 py-6 gap-3">
+                Date Added
+              </th>
+              <th className="hidden md:table-cell text-custom-color2 text-sm font-normal leading-5 px-3 py-6 gap-3">
+                Date Deleted
+              </th>
+              <th className="hidden md:table-cell text-custom-color2 text-sm font-normal leading-5 px-3 py-6 gap-3">
+                Status
+              </th>
+              <th className="hidden lg:table-cell text-custom-color2 text-sm font-normal leading-5 px-3 py-6 gap-3">
+                Action
+              </th>
+            </tr>
+          </thead>
+          <tbody>
+            {/* Listed Products */}
+            {deletedProducts.map((product, index) => (
+              <tr className="border-t border-custom-color1" key={index}>
+                <td className="text-xs tracking-wider lg:tracking-wide font-manropeL lg:text-base text-custom-color10 px-6 py-4 items-center gap-6 self-stretch flex">
+                  <input type="checkbox" />
+
+                  {product.name}
+                </td>
+                <td className="text-xs tracking-wider lg:tracking-wide font-manropeL lg:text-base text-custom-color2 px-6 py-4 text-center">
+                  {product.vendor}
+                </td>
+                <td className="hidden md:table-cell text-xs tracking-wider lg:tracking-wide font-manropeL lg:text-base text-custom-color2 px-6 py-4 text-center">
+                  #{product.id}
+                </td>
+                <td className="hidden md:table-cell text-xs tracking-wider lg:tracking-wide font-manropeL lg:text-base text-custom-color2 px-6 py-4 text-center">
+                  {product.dateAdded.getDate().toString().padStart(2, '0')}-
```

Comment:
the javscript native method, **_new Intl.DateTimeFormat(),_** is a better way to handle date formatting